src/currencyservice/client.js

```diff
@@ -31,8 +31,11 @@ const client = new shopProto.CurrencyService(`localhost:${PORT}`,
 const logger = pino({
   name: 'currencyservice-client',
   messageKey: 'message',
-  changeLevelName: 'severity',
-  useLevelLabels: true
+  formatters: {
+    level (logLevelString, logLevelNum) {
+      return { severity: logLevelString }
+    }
+  }
```

Comment:
Just the clarify: This `level` is a function. `pino` passes in two arguments when it's called.